### ISSUE TITLE
Edit Trip Details layout on iPhone needs work

### ISSUE THREAD

**Issue by EVGrokker** [aesculus/EVTO-App-Feedback#315]
v 1.2 (2)

The new Edit Trip Details dialog looks wrong on the iPhone. It appears that the width of the Edit Trip Details dialog is less than the width of the Edit Segment Details dialog, whose layout is the reference.

Also, I believe that the design decision was to remove the Comments field for Trips.

---

![edit trip details](https://cloud.githubusercontent.com/assets/26678403/25333494/aa323486-28a7-11e7-89a4-92eda79783dd.PNG)

**Comment by aesculus** [comment]
Please test again in V1.2 (23)

**Comment by EVGrokker** [comment]
Here's a suggested design for adding a checkbox enabling/disabling HVAC, corresponding to a similar control in Tab 2 (cockpit settings) of the My Cars dialog as discussed in #187.

![edittripdetails](https://user-images.githubusercontent.com/26678403/26994509-252e8f62-4d25-11e7-8203-853621b78e4d.jpg)

**Comment by EVGrokker** [comment]
Note that toggling HVAC to OFF state should gray out and disable editing temperature setting.

**Comment by EVGrokker** [comment]
Here's a proposed revision to the layout, adding a picker for the car assigned to the trip.

Notes:
- I removed the 'Name:' line to create some extra space. 
- There's no significance to the narrower car picker in the mockup, I just grabbed it from another image. It should be the same width as the text edit field below it.

![edittripdetails copy](https://user-images.githubusercontent.com/26678403/27003757-eb0b5c58-4db9-11e7-9b1c-5ba07528f7f2.jpg)
